supabase/migrations/20230329202715_add_project_columns.sql

```diff
@@ -0,0 +1,28 @@
+alter table public.project_subscriptions 
+drop COLUMN "project_id";
+
+alter table public.projects 
+alter COLUMN "id" type text,
+add COLUMN "handle" text,
+add COLUMN "projectId" int,
+add COLUMN "pv" char(1),
```

Comment:
**suggestion** this could be an enum of `1` and `2`

Reply:
tried this but i don't think enum is an option for server types?